spec/support/acts_as_tenant.rb

```diff
@@ -9,10 +9,12 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        @old_host = Capybara.app_host
```

Comment:
For example, If you have something like this <code> visit course_achievements_path(course)</code>, you will see the exception of cannot find the course, because my course was under the instance I created, not default.